### PR TITLE
cancel pci eeh test if system is no ppc

### PR DESCRIPTION
**Pull request by fkromer**
closes GH-716

### CODE REVIEW

**Comment by fkromer** [comment]
Seems like the PR failed because the [commit message is not long enough](https://travis-ci.org/avocado-framework-tests/avocado-misc-tests/builds/309547250#L558). From a functional point of view it works just fine on a system with ARMv7 processor.

**Comment by vrbagalkote** [comment]
@fkromer correct from coding point it works fine, but there is a format which travis expect. Just to make travis happy, send the patch in below format.

        echo "ERR: Commit message is too short (less than 5 lines)."

        echo "     Here a minimal template:"

        echo "     ------------------------------------"

        echo "     header          <- Limited to 50 characters. No period."

        echo "                     <- Blank line"

        echo "     message         <- Any number of lines, limited to 72 characters per line."

        echo "                     <- Blank line"

        echo "     Signed-off-by:  <- Signature (created by git commit -s)"

        echo "     ------------------------------------"

File: io/pci/PowerNVEEH.py [hunk 1]
```diff
@@ -54,6 +54,9 @@ def setUp(self):
         """
         Gets the console and set-up the machine for test
         """
+        cmd = "uname -p"
+        if 'ppc' not in process.system_output(cmd, ignore_status=True):
```
Comment (harish-24) [inline io/pci/PowerNVEEH.py]: Please make use of the avocado library `from avocado.utils import distro`
```
if 'ppc' not in distro.detect().arch:
    self.cancel("Processor is not ppc64")
```

File: io/pci/PowerVMEEH.py [hunk 2]
```diff
@@ -56,6 +56,9 @@ def setUp(self):
         """
         Gets the console and set-up the machine for test
         """
+        cmd = "uname -p"
+        if 'ppc' not in process.system_output(cmd, ignore_status=True):
```
Comment (harish-24) [inline io/pci/PowerVMEEH.py]: here too

**Comment by PraveenPenguin** [comment]
@fkromer one more request please squash all three commit in single commit

File: io/pci/PowerNVEEH.py (hunk 1 above)
Comment (fkromer) [reply io/pci/PowerNVEEH.py]: @harish-24 Done

File: io/pci/PowerVMEEH.py (hunk 2 above)
Comment (fkromer) [reply io/pci/PowerVMEEH.py]: Done

**Comment by fkromer** [comment]
@PraveenPenguin should be ok now

**Comment by PraveenPenguin** [comment]
@vrbagalkote @harish-24 thanks for review

**Pull request merged**